import-service/serverless.ts

```diff
@@ -99,6 +116,15 @@ const serverlessConfiguration: Serverless = {
                 },
               },
             },
+            cors: true,
+            authorizer: {
+              name: "tokenAuthorizer",
+              arn:
+                "arn:aws:lambda:eu-west-1:428742678759:function:authorization-service-dev-basicAuthorizer",
```

Comment:
It is not a good practice to expose your account id. Event in AWS documentation it is mentioned that account id is sensitive and secure information that should not be shared if it possible

But everything else looks fine for me

Reply:
Thanks!
Implemented it "just for test" and forgot to remove it.

Fixed.